mkdocs.yml

```diff
@@ -53,10 +53,10 @@ nav:
     - OpenAPI Extension: reference/extension.md
     - Architecture: reference/architecture.md
     - Dashboard API: reference/openapi.md
-    - Analytics: reference/analytics.md
   - Contributing:
     - Development: 'contributing/development.md'
     - Roadmap: 'contributing/roadmap.md'
+  - Privacy Policy: privacy.md
```

Comment:
What's up the indentation here? Also, small point, can you follow the structure of the surrounding block by having a single quote (') around the file.